src/main/webapp/resources/script/getHost.js

```diff
@@ -0,0 +1 @@
+document.getElementById("REST").innerHTML = "REST API: $ curl  '"+window.location.protocol+"//"+window.location.host+"/api'";
```

Comment:
I suggest instead of saying 

> REST API:

=>
Command Version:

Reply:
I think

“Cmdline version”

would be even better